sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/conditionalExpressions.scala

```diff
@@ -206,6 +212,25 @@ case class CaseWhen(branches: Seq[Expression]) extends CaseWhenLike {
       case Seq(elseValue) => s" ELSE $elseValue"
     }.mkString
   }
+
+  override def sql: Option[String] = {
+    sequenceOption(branches.map(_.sql)).map {
+      case branchesSQL =>
```

Comment:
why "case" here? maybe put this on the previous line to save one level of indentation.